vision/benchmarks/accuracy/bench_module.py

```diff
@@ -352,26 +352,33 @@ def bench_module_accuracy(folder: str, quiet_output: bool = False, draw_centers:
                         "CRASH:",crash,
                     )
                 
+
+                marked_image = np.copy(image)
+
                 # if enabled and available, draw circle on center
                 if draw_centers:
-                    image_copy = np.copy(image)
                     cv2.circle(
-                        image_copy, 
-                        (center[0], center[1]), 
-                        20, 
-                        (0, 0, 255), 
-                        3
+                        img=marked_image, 
+                        center=(center[0], center[1]), 
+                        radius=20, 
+                        color=(0, 0, 255), 
+                        thickness=3
                     )
                     cv2.circle(
-                        image_copy, 
-                        (center[0], center[1]), 
-                        1, 
-                        (0, 0, 255), 
-                        2
+                        img=marked_image, 
+                        center=(center[0], center[1]), 
+                        radius=1, 
+                        color=(0, 0, 255), 
+                        thickness=2
                     )
 
-                    cv2.imwrite(os.path.join(DRAW_CENTERS_DIR,file), image_copy)
+                # if enabled and available, draw hough circles
+                if draw_hough_circles and tester.location.circles is not None:
+                    for x, y, r in tester.location.circles:
+                        cv2.circle(marked_image, (x, y), r, (0, 255, 0), 4)
+                        cv2.rectangle(marked_image, (x - 5, y - 5), (x + 5, y + 5), (0, 128, 255), -1)
 
+                cv2.imwrite(os.path.join(OUTPUT_IMGS_DIR,file), marked_image)
```

Comment:
If you add the if statement I suggested above, this would also need an if statement.